package.json

```diff
@@ -18,7 +18,7 @@
     "api"
   ],
   "dependencies": {
-    "unirest": "*"
```

Comment:
This is very dangerous and can break any time.
https://github.com/Mashape/unirest-nodejs/pull/41